docs/index.bs

```diff
@@ -1986,78 +1983,64 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
-                1. Set the {{CacheBatchOperation/type}} dictionary member of |o| to "put".
-                1. Set the {{CacheBatchOperation/request}} dictionary member of |o| to |requestArray|[|index|].
-                1. Set the {{CacheBatchOperation/response}} dictionary member of |o| to |response|.
-                1. Add |o| to |operations|.
-            1. Let |resultPromise| be the result of running <a>Batch Cache Operations</a> algorithm passing |operations| as the argument.
-            1. Return the result of <a>transforming</a> |resultPromise|  with a fulfillment handler that, when called with argument |responses|, performs the following substeps <a>in parallel</a>:
-                1. Let |responseBodyPromiseArray| be an empty array.
-                1. For each |response| in |responses|:
-                    1. Let |responseBodyPromise| be a new <a>promise</a>.
-                    1. Run the following substeps <a>in parallel</a>:
-                        1. Wait for either end-of-file to have been pushed to |response|'s associated [=Response/response=] |r|'s [=response/body=] or for |r| to have a [=response/termination reason=].
-                        1. If |r| had a [=response/termination reason=], then:
-                            1. If the <a>incumbent record</a> |incumbentRecord| of the corresponding <a>fetching record</a> |fetchingRecord| in <a>request to response map</a> is not null, then:
-                                1. Set |fetchingRecord| in <a>request to response map</a> to the copy of |incumbentRecord|.
-                            1. Else:
-                                1. Delete |fetchingRecord| from <a>request to response map</a>.
-                            1. Reject |responseBodyPromise| with a <code>TypeError</code>.
-                        1. Else:
-                            1. Set the <a>incumbent record</a> of the corresponding <a>fetching record</a> |fetchingRecord| in <a>request to response map</a> to the copy of |fetchingRecord|.
-                            1. Let |invalidRecords| be the result of running <a>Query Cache</a> algorithm passing |fetchingRecord|.\[[key]] as the argument.
-                            1. For each |invalidRecord| in |invalidRecords|:
-                                1. If |invalidRecord| is not |fetchingRecord|, delete it from <a>request to response map</a>.
-                            1. Resolve |responseBodyPromise| with |response|.
-                    1. Add |responseBodyPromise| to |responseBodyPromiseArray|.
-                1. Let |q| be <a>waiting for all</a> of |responseBodyPromiseArray|.
-                1. Return the result of <a>transforming</a> |q|  with a fulfillment handler that returns undefined.
+                        1. [=list/For each=] |fieldValue| of |fieldValues|:
+                            1. If |fieldValue| matches "`*`", then:
+                                1. Set |matchAsterisk| to true.
+                                1. [=Break=].
```

Comment:
Could we just reject and abort steps here? Means we don't need *matchAsterisk*.

Reply:
The same thought/discussion as above.

Reply:
This one is slightly different since it's in parallel steps.

Right now we're waiting on fetches to complete even though we know the operation is a failure, so it seems better to reject *responsePromise*, abort the ongoing fetches, then abort the steps.

Reply:
Addressed.